src/components/forms/PlanForm/index.tsx

```diff
@@ -1150,6 +1150,7 @@ const PlanForm = (props: PlanFormProps) => {
                                 className="btn-light btn-block"
                                 id={`plan-trigger-conditions-${index}`}
                                 hidden={hiddenFields.includes('triggersAndConditions')}
```

Comment:
@ciremusyoka have you consider conditional rendering?

Reply:
@machariamuguku The form has to be populated but hidden.